src/main.rs

```diff
@@ -12,6 +20,11 @@ fn cli() -> Command {
                 .subcommand(
                     Command::new("l1")
                         .about("Read state from Ethereum L1")
+                        .arg(
+                            arg!(--"http-url" <HTTP_URL>)
+                                .help("Ethereum JSON-RPC HTTP URL")
+                                .default_value("https://eth.llamarpc.com"),
```

Comment:
```suggestion
```
I don't think we should provide a default actually, we could mention it in the readme though.